fork-runner/src/main/java/com/shazam/fork/PermissionUtils.java

```diff
@@ -0,0 +1,48 @@
+package com.shazam.fork;
+
+import com.android.ddmlib.*;
+import com.shazam.fork.model.Permission;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.util.List;
+
+import static java.lang.String.format;
+
+public class PermissionUtils {
```

Comment:
If we change from static, maybe rename this (won't be a *Utils class at that point).

Reply:
👍